src/assets/fonts/fonts.css

```diff
@@ -0,0 +1,24 @@
+@font-face{
+    font-family: 'Open Sans';
+    src: url('./OpenSans-Regular.ttf');
+    font-style: normal;
+    font-weight: 400;
+  }
+  @font-face{
+    font-family: 'Open Sans SemiBold';
+    src: url('./OpenSans-SemiBold.ttf');
```

Comment:
File fonts.css  should be in assets/css directory and only import from assets/fonts fonts file.